src/signature.js

```diff
@@ -3,24 +3,30 @@ const crypto = require('crypto');
 
 class Signature {
 
-	constructor(partnerID, apiKey) {
-		this.partnerID = partnerID;
-		this.apiKey = apiKey;
-	}
+  constructor(partnerID, apiKey) {
+      this.partnerID = partnerID;
+      this.apiKey = apiKey;
+  }
 
-	generate_sec_key(timestamp=Date.now()) {
-    var hash = crypto.createHash('sha256').update(parseInt(this.partnerID, 10) + ":" + timestamp).digest('hex');
-    var encrypted = crypto.publicEncrypt({
+  /**
+   * @deprecated Use generate_signature instead
+   */
+  generate_sec_key(timestamp=Date.now()) {
```

Comment:
We removed these in the other libraries so let's remove them here. We should strive to maintain as much consistency across our libraries as it lowers the cognitive requirements of using and maintaining them